mingw-w64-ffmpeg/PKGBUILD

```
@@ -121,6 +120,13 @@ build() {
     --enable-zlib
     --disable-doc
   )
+
+  if [  ${MINGW_PACKAGE_PREFIX} == *-clang-*  ]; then
```

isn't this the other way around?

---

You'd think that, but it got around the compiling issues lol. I basically lifted the idea from the qt5 package

---

maybe try leaving this change out and keeping `--enable-frei0r` in the main block of common_config?

---

because it's the command line option that forces the check for frei0r headers and errors out, it can't be left, it needs that change.

---

It was already enabled before for clang, so it must have been spurious failure/success.

---

> It was already enabled before for clang, so it must have been spurious failure/success.

Kept failing for me locally and in a VM, I mean I'll make the change, but I'm pretty sure it'll fail.

---

doesn't make any sense though

---

Ohhh,

```suggestion
  if [[  ${MINGW_PACKAGE_PREFIX} != *-clang-*  ]]; then
```

Gotta have double brackets for the pattern matching stuff to work.